### ISSUE TITLE
Error when using RequireJS loader

### ISSUE THREAD

**Issue by mweaver** [theintern/intern#115]
When I try to run 'client.js' from the command line using the RequireJS loader (via the 'useLoader' setting in the config file), I get the following error:

```
$ node node_modules/intern/client.js config=tests/intern

<omitted_path>/node_modules/intern/node_modules/dojo/node.js:5
        throw new Error('Cannot find the Node.js require');
          ^
Error: Cannot find the Node.js require
    at <omitted_path>/node_modules/intern/node_modules/dojo/node.js:5:9
    at Object.context.execCb (<omitted_path>/node_modules/intern/node_modules/requirejs/bin/r.js:1876:33)
    at Object.Module.check (<omitted_path>/node_modules/intern/node_modules/requirejs/bin/r.js:1110:51)
    at Object.Module.enable (<omitted_path>/node_modules/intern/node_modules/requirejs/bin/r.js:1381:22)
    at Object.Module.init (<omitted_path>/node_modules/intern/node_modules/requirejs/bin/r.js:1018:26)
    at callGetModule (<omitted_path>/node_modules/intern/node_modules/requirejs/bin/r.js:1408:63)
    at Object.context.completeLoad (<omitted_path>/node_modules/intern/node_modules/requirejs/bin/r.js:1770:21)
    at Function.req.load (<omitted_path>/node_modules/intern/node_modules/requirejs/bin/r.js:2525:17)
    at Object.context.load (<omitted_path>/node_modules/intern/node_modules/requirejs/bin/r.js:1865:21)
    at Object.Module.load (<omitted_path>/node_modules/intern/node_modules/requirejs/bin/r.js:1064:29)
```

When I take a closer look at the 'dojo/node.js' file near line 5, I notice in comments that it is expecting the node.js loader to be exposed via 'this.require.nodeRequire'.  When using the default dojo leader, this is indeed the case.

However, when using RequireJS, the 'nodeRequire' is buried one level further in at: 'this.requrejsVars.require.nodeRequire'.

So, I don't know if that's a bug or if I'm doing something else wrong, but if you could take a look at this, I would certainly appreciate it.

I've also created a git repo that contains a project that illustrates this problem.  If you're interested, you can clone it from -> https://github.com/mweaver/interniotest.git

Thank you for your time!

**Comment by deeperx** [comment]
I've ```intern 3.0.0``` and when I try to run functional tests it says:

```Error: Error: Cannot find the Node.js require at http://localhost:8000/node_modules/intern/node_modules/dojo/node.js:14:59```

Is this the same problem or not?

**Comment by mweaver** [comment]
Yes, I believe it is.

**Comment by csnover** [comment]
@deeperx No, you are trying to load a Node.js module in a browser which you can’t do for hopefully obvious reasons :)

**Comment by deeperx** [comment]
@csnover actually, I'm accessing the localhost inside a server not in a browser code directly. And, intern does that not me :)

**Comment by benlangfeld** [comment]
I see this issue with Intern 3.0.0:

```
➭ grunt intern:unit
Running "intern:unit" (intern) task
Listening on 0.0.0.0:9000
Tunnel started
‣ Created session phantomjs on any platform (51e58e3b-ce2f-4860-8743-13f525896e92)
>> Suite phantomjs on any platform - unit tests FAILED
Error: Error: Cannot find the Node.js require at http://192.168.88.1:9000/__intern/node_modules/dojo/node.js:14:59
No stack or location
× phantomjs on any platform - Basic chat operation - should load the correct page (0.237s)
AssertionError: expected 'Browser test runner' to equal 'Candy - Chats are not dead yet'
  at Assertion.assertEqual  <node_modules/intern/node_modules/chai/chai.js:733:12>
  at Assertion.ctx.(anonymous function) [as equal]  <node_modules/intern/node_modules/chai/chai.js:3475:25>
  at Command.<anonymous>  <tests/candy/functional/basics.js:20:23>
  at runCallback  <node_modules/intern/node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:543:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/node_modules/dojo/nextTick.ts:44:3>
  at process._tickCallback  <node.js:355:11>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at Test.test  <tests/candy/functional/basics.js:19:6>
  at <node_modules/intern/lib/Test.js:211:24>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/node_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:317:4>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/node_modules/dojo/nextTick.ts:44:3>
  at process._tickCallback  <node.js:355:11>
No unit test coverage for phantomjs on any platform
>> phantomjs on any platform: 1/1 tests failed


>> TOTAL: tested 1 platforms, 1/1 tests failed
Warning: Test failure; check output above for details. Use --force to continue.

Aborted due to warnings.
```

I don't use NodeJS `require()` right now, and it seems neither does Chai at that line number, so I am utterly confused.

**Comment by benlangfeld** [comment]
Has anyone seen the above issue before? This is, unfortunately, preventing Candy from upgrading Intern and using Webpack.

**Comment by csnover** [comment]
@benlangfeld This ticket is for a bug that was fixed in Intern 1.3.2 and doesn’t have anything to do with the issue you are describing, which is #491 and is fixed in Intern 3.0.3. Thanks!

**Comment by benlangfeld** [comment]
That seems to have fixed it. Thanks @csnover!
